app/src/main/res/navigation/nav_graph.xml

```diff
@@ -82,7 +82,7 @@
         </fragment>
         <fragment
             android:id="@+id/loginFragment"
-            android:name="org.greenstand.android.TreeTracker.fragments.LoginFragment"
+            android:name="org.greenstand.android.TreeTracker.login.LoginFragment"
```

Comment:
There is a second navGraph you can use instead of this one. This is for the legacy app flow, I'd like to keep them in parallel until the new UI is complete.

Reply:
Fantastic! I was unaware of this. Will fix